lib/components/viewers/stop-schedule-table.js

```diff
@@ -118,7 +126,7 @@ class StopScheduleTable extends Component {
             const refProp = scrollToRow ? this.targetDepartureRef : null
             return (
               <tr key={index} className={className} ref={refProp}>
-                <td>{blockId}</td>
+                <BlockCell>{blockId}</BlockCell>
```

Comment:
@binh-dam-ibigroup, since this is using ellipsis overflow, we should add `title={blockId}` to the cell so that on hover a user can view the full value.

Reply:
Good point. Updated in https://github.com/opentripplanner/otp-react-redux/pull/292/commits/cdf913d9e17f192608e8212a306f436e6680085f.